src/ui/components/Onboarding/index.jsx

```diff
@@ -1,5 +1,144 @@
+import React, { useState, useEffect } from "react";
+import { useNavigate } from "react-router-dom";
+import styled from "styled-components";
+
+import Button from "../shared/Button";
+import ToastPopup from "../shared/Toast";
+
+import useProjectVersionStore from "../../../store/projectVersion";
+import useProjectStore from "../../../store/project";
+
+import postMessage from "../../../utils/postMessage";
+import getAllVersions from "../../../services/getAllVersions";
+import figciLogo from "../../../../assets/logo_figci.png";
+import onboarding from "../../../../assets/onboarding.png";
+
 function Onboarding() {
-  return <h1>onboarding</h1>;
+  const navigate = useNavigate();
+  const [projectInformation, setProjectInformation] = useState({});
+  const [toast, setToast] = useState("");
```

Comment:
- `P2`
```suggestion
  const [toast, setToast] = useState({});
```

`toast`또한 할당될 type으로 초깃값을 설정해주시면 더 명시적이지 않을까 생각합니다!